packages/controlled-form/src/lib/OrganizationAutocomplete.tsx

```diff
@@ -35,14 +39,14 @@ export const ControlledOrganizationAutocomplete = ({
         shouldUnregister,
         validate,
         value,
+        ...rules,
       }}
       shouldUnregister={shouldUnregister}
-      render={({ field: { onChange, value, onBlur }, fieldState: { error } }) => (
+      render={({ field: { onChange, value, onBlur, ref }, fieldState: { error } }) => (
         <OrganizationAutocomplete
           {...rest}
           FieldProps={{
             ...FieldProps,
-            required: typeof required === 'object' ? required.value : !!required,
```

Comment:
Removing this broke the asterisk. It is needed to indicate to the underlying TextField that it is required and to show the asterisk.